common/lib/src/realm_common_base.dart

```diff
@@ -57,10 +57,12 @@ class RealmModel {
   const RealmModel([this.type = ObjectType.realmObject]);
 }
 
-/// MapTo annotation for class member level.
+/// MapTo annotation for class level and class member level.
 ///
-/// Indicates that a property should be persisted under a different name.
-/// This is useful when opening a Realm across different bindings where code style conventions might differ.
+/// MapTo on class level indicates that a class should be persisted under a different collection name.
```

Comment:
```suggestion
/// MapTo on class level indicates that a class should be persisted under a different name.
```
The word `collection` here is not clear. I suggest removing it

Reply:
I applied your next suggestion.